doc/flame/inputs/drag_events.md

```diff
@@ -1,8 +1,13 @@
 # Drag Events
 
 ```{note}
-This document describes the new drag events API. The old (legacy) approach,
-which is still supported, is described in [](gesture_input.md).
+This document describes the new drag events API. It is important to note that 
+when the first draggable component is added to the game, `FlameGame` must 
+temporarily remove itself and run the `GestureDetectorBuilder` which will add a 
+gesture detector as the first child to `FlameGame`, and then re-add itelf.  As 
+such, `FlameGame`'s `onRemove` event will be triggered.  A simple boolean flag 
```

Comment:
I think these probably should be two notes, or it's own section, now the next is in the middle of the previous note and reads a bit strange.

Reply:
I thought about that, but thought two notes at the top looked weird.  I was wanting to ride on the note about the new API, but I can put it down below.